test/integration/users.test.js

```diff
@@ -585,6 +574,48 @@ describe("Users", function () {
     });
   });
 
+  describe("GET /users/?id", function () {
+    it("Should return given user by id", function (done) {
+      chai
+        .request(app)
+        .get(`/users/?id=${userId}`)
+        .end((err, res) => {
+          if (err) {
+            return done(err);
+          }
+          expect(res).to.have.status(200);
+          expect(res.body).to.be.a("object");
+          expect(res.body.message).to.equal("User returned successfully!");
+          expect(res.body.user).to.be.a("object");
+          expect(res.body.user).to.have.property("username");
+          expect(res.body.user).to.have.property("first_name");
+          expect(res.body.user).to.have.property("last_name");
+          expect(res.body.user).to.have.property("yoe");
+          expect(res.body.user).to.have.property("linkedin_id");
+          expect(res.body.user).to.have.property("github_id");
+          expect(res.body.user).to.have.property("isMember");
+          expect(res.body.user).to.have.property("roles");
```

Comment:
can you try to achieve this in a single expect statement ?

Reply:
done.